### PR TITLE
RDOIAT-374: Improve serialization performance

### PR DESCRIPTION
**Pull request by cangosta**

### CODE REVIEW

File: CefGlue.Tests/Serialization/JavascriptToNativeTypeConverterTests.cs [hunk 1]
```diff
@@ -24,6 +24,7 @@ public void BasicTypes()
             Assert.AreEqual(TestEnum.OptionB, JavascriptToNativeTypeConverter.ConvertToNative<TestEnum>(1));
 
             Assert.AreEqual("this is a string", JavascriptToNativeTypeConverter.ConvertToNative<string>("this is a string"));
+            Assert.AreEqual("", JavascriptToNativeTypeConverter.ConvertToNative<string>(""));
```
Comment (joaompneves) [inline CefGlue.Tests/Serialization/JavascriptToNativeTypeConverterTests.cs:27]: we need a test for the scenario fixed:

similar to this:
https://github.com/OutSystems/CefGlue/blob/ide/task/RDOIAT-374-improve-serialization/CefGlue.Tests/Javascript/NativeObjectInteropTests.cs#L147

File: CefGlue.BrowserProcess/Serialization/V8ValueSerialization.cs [hunk 2]
```diff
@@ -39,16 +44,11 @@ private static void SerializeV8Object(CefV8Value obj, CefValueWrapper cefValue,
             }
             else if (obj.IsString)
             {
-                cefValue.SetString(obj.GetStringValue());
+                CefValueSerialization.Serialize(obj.GetStringValue() ?? "", cefValue);
```
Comment (joaompneves) [inline CefGlue.BrowserProcess/Serialization/V8ValueSerialization.cs:47]:
```suggestion
                // default to "", because cef converts "" to null, and when null it will fall on the Null case
                CefValueSerialization.Serialize(obj.GetStringValue() ?? "", cefValue);
```

**Review by joaompneves — CHANGES_REQUESTED** [review]

File: CefGlue.Tests/Javascript/NativeObjectInteropTests.cs [hunk 3]
```diff
@@ -162,6 +169,22 @@ public async Task MethodParamsArePassed()
             Assert.AreEqual(DateTime.Parse(Date), result[2]);
             Assert.AreEqual(true, result[3]);
         }
+        
+        [Test]
+        public async Task MethodEmptyStringParamIsPassed()
```
Comment (joaompneves) [inline CefGlue.Tests/Javascript/NativeObjectInteropTests.cs:174]: please also test passing null

**Review by joaompneves — DISMISSED** [review]

**Review by joaompneves — APPROVED** [review]
